.github/workflows/build.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         if: steps.cache-nuget.outputs.cache-hit != 'true'
 
       - name: 🧰 Setup BuildToolkit
-        uses: moryx-industry/setup-buildtoolkit@v1
+        uses: moryx-industry/setup-buildtoolkit@fix/reuse-wwwroot
```

Comment:
do you intentionally refer to the branch here, or do you first need the other branch to be merged to clean this up? 🤔

Reply:
It required the latest changes in `setup-buildtoolkit` to be released